relay-general/src/store/normalize/breakdowns.rs

```diff
@@ -202,15 +202,19 @@ impl EmitBreakdowns for SpanOperationsConfig {
 
 /// Configuration to define breakdown to be generated based on properties and breakdown type.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-#[serde(tag = "type", rename_all = "snake_case")]
+#[serde(tag = "type", rename_all = "camelCase")]
 pub enum BreakdownConfig {
```

Comment:
@untitaker I couldn't really use `enum ErrorBoundary<T>` in the project config (https://github.com/getsentry/relay/blob/a231a2b17a71304a19710cf9a49b21fa06e25f71/relay-general/src/store/mod.rs#L55) as it's in a different crate (https://github.com/getsentry/relay/blob/a231a2b17a71304a19710cf9a49b21fa06e25f71/relay-server/src/utils/error_boundary.rs).

I'm assuming this is what you had in mind. 

If so, we might need to consolidate `enum ErrorBoundary<T>` somewhere, and implement `Default` for `enum ErrorBoundary<T>`.